var/spack/repos/builtin/packages/py-psutil/package.py

```diff
@@ -15,12 +15,17 @@ class PyPsutil(PythonPackage):
     url      = "https://pypi.io/packages/source/p/psutil/psutil-5.6.3.tar.gz"
 
     version('5.6.3', sha256='863a85c1c0a5103a12c05a35e59d336e1d665747e531256e061213e2e90f63f3')
+    version('5.6.2', sha256='828e1c3ca6756c54ac00f1427fdac8b12e21b8a068c3bb9b631a1734cada25ed')
     version('5.5.1', sha256='72cebfaa422b7978a1d3632b65ff734a34c6b34f4578b68a5c204d633756b810')
     version('5.4.5', sha256='ebe293be36bb24b95cdefc5131635496e88b17fabbcf1e4bc9b5c01f5e489cfe')
     version('5.0.1', sha256='9d8b7f8353a2b2eb6eb7271d42ec99d0d264a9338a37be46424d56b4e473b39e')
 
+    variant('enum', default=False)
```

Comment:
I'm wondering about this. If we do want to keep it, let's add a description. We could also just always add it. It's unclear if it's meant as an additional optional feature, or a required feature for older versions.

Reply:
TBH, I couldn't figure out what this option does (aside from adding dependency). It looks like the author planned to use enums with all versions, but forgot/changed mind. I'll comment it out, since it does nothing.

Reply:
TBH, I couldn't figure out what this option does (aside from adding dependency). It looks like the author planned to use enums with all versions, but forgot/changed mind. I'll comment it out, since it does nothing.